jupyterlab/labbox_ephys_widgets_jp/labbox_ephys_widgets_jp/create_sorting_view.py

```diff
@@ -37,6 +37,8 @@ class SortingView(DOMWidget):
         recordingObject = DictTrait(recording.object()).tag(sync=True)
         recordingInfo = DictTrait(le.get_recording_info(recording_object=recording.object())).tag(sync=True)
         sortingInfo = DictTrait(le.get_sorting_info(sorting_object=sorting.object(), recording_object=recording.object())).tag(sync=True)
+        selection = DictTrait({}).tag(sync=True)
```

Comment:
Add new selection and curation traits to the jupyter python object. This will allow:

X = sorting_views.MVSortingView(...)
X.curation = set curation here

Or retrieve the curation in a different cell:
X.curation